manager/assets/modext/core/modx.js

```diff
@@ -347,13 +347,30 @@ Ext.extend(MODx,Ext.Component,{
         }
 
         if (MODx.config["static_elements_automate_" + typePlural] == 1) {
-            if (Ext.getCmp("modx-" + type + "-category").getValue() > 0) {
-                category = Ext.getCmp("modx-" + type + "-category").lastSelectionText;
+            category = Ext.getCmp("modx-" + type + "-category").getValue();
+            if (category > 0) {
+                Ext.Ajax.request({
+                    url: MODx.config.connector_url,
+                    params: {
+                        action: 'Element/Category/GetList',
+                        id: category,
+                        limit: 0,
+                    },
+                    success: function (response) {
+                        var data = Ext.decode(response.responseText);
+                        categoryText = (data && data.success && data.results) ? data.results[0].name : '';
```

Comment:
```suggestion
                        categoryText = data && data.success && data.results ? data.results[0].name : '';
```

I don't see the point of grouping the left operand

Reply:
It's easier to see the condition part of the ternary, if it is surrounded by braces.